applications/nrf_desktop/src/modules/Kconfig.ble_scan

```diff
@@ -55,6 +55,13 @@ config DESKTOP_BLE_SCAN_KEYBOARD_LIMIT
 	default 1
 	range 0 255
 
+config DESKTOP_BLE_NEW_PEER_SCAN_ON_BOOT
```

Comment:
It would be great to keep the same prefix DESKTOP_BLE_SCAN to indicate clearly to which module a given configuration belongs to

Reply:
`NCSDK-19652`